pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -332,14 +382,137 @@ servingInfo:
   keyFile: /var/serving-cert/tls.key
 providers: 
   statuspageID: id-1234
+`,
+				},
+			},
+		},
+		{
+			name: "Test operator config with Statuspage pageID",
```

Comment:
I believe we already had this test, trying to figure out why its showing up as new.